modules/overview-of-user-types-and-permissions.adoc

```diff
@@ -69,6 +70,7 @@ There are some operations relevant to {productname-short} that require the `clus
 * Adding users to the {productname-short} user and administrator groups, if you are using specialized groups.
 * Removing users from the {productname-short} user and administrator groups, if you are using specialized groups.
 * Managing custom environment and storage configuration for users in {openshift-platform}, such as Jupyter notebook resources, ConfigMaps, and persistent volume claims (PVCs).
+* Creating a data science pipeline server and pipeline from the {productname-short} user interface or from the command-line interface (CLI), for any data science project. 
```

Comment:
@chtyler , this sentence is not correct. You don't need to be cluster-admin to create a data science pipeline server or import a pipeline

Reply:
Sentence removed.